osu.Game/Migrations/20181007180454_StandardizePaths.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System.IO;
+
+namespace osu.Game.Migrations
+{
+    public partial class StandardizePaths : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            string sanitized = Path.DirectorySeparatorChar.ToString();
+            string standardized = "/";
+
+            // If we are not on Windows, no changes are needed.
+            if (string.Equals(standardized, sanitized, StringComparison.Ordinal))
+                return;
```

Comment:
Isn't this going to fail on unix if any of the paths below already have a `\` in the database? Probably want to unconditionally run the up/down migration.

Reply:
also, the `Down` migration should probably just be removed, as it's not guaranteed to restore to the "previous" state anyway (and the previous state was 100% broken so restoring isn't ever going to be something we want to do).

Reply:
Unix systems shouldn't have Windows-style path separators in their database, all user input gets sanitized at this point, but I've removed the check and downwards migration.

Reply:
Well some inputs like the audio and background files were not sanitized, so those are potentially going to still be broken.